src/internal/mocks/referencePointMocks.ts

```diff
@@ -1796,3 +1861,50 @@ export const twoMeasuresOneCategoryWithStackMeasuresToPercent: IReferencePoint =
         },
     },
 };
+
+export const measureValueFilterReferencePoint: IReferencePoint = {
+    buckets: [
+        {
+            localIdentifier: "measures",
+            items: masterMeasureItems,
+        },
+        {
+            localIdentifier: "view",
+            items: attributeItems,
+        },
+    ],
+    filters: {
+        localIdentifier: "filters",
+        items: [
+            {
+                localIdentifier: "fbv1",
+                filters: [
+                    {
+                        measureLocalIdentifier: masterMeasureItems[0].localIdentifier,
+                        condition: {
+                            comparison: {
+                                operator: "GREATER_THAN",
+                                value: 100,
+                            },
+                        },
+                    },
+                ],
+            },
+            {
+                localIdentifier: "fbv2",
```

Comment:
Can you please add one more filter without condition? Just to be sure that it is covered when someone will refactor it.